### PR TITLE
update json file if already exist and add comments

### PR DESCRIPTION
**Pull request by ngdream**
## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,11 +25,20 @@ function start() {
       if (fs.existsSync(`./public/data/${GHUsername}.json`)) {
         console.log(
           chalk.bgYellow.bold(
-            ` File with ${answers.name}.json already exists, try again! `
+            ` File with ${answers.name}.json already exists, your data will be updated ! `
```
Comment (schmelto) [inline src/index.js:28]:
```suggestion
            ` File with ${answers.name}.json already exists, your data will be updated! `
```

**Comment by Pradumnasaraf** [comment]
Sorry, I missed this PR. Will test and check this weekend. Thnak you!

**Review by Pradumnasaraf — COMMENTED** [review]
Thanks for the PR.

I just tested it out and found a catch, that it's prompting `creating....` even if the user with that user is not found on GitHub. 


Screenshot from 2022-09-25 19-23-22

**Comment by Pradumnasaraf** [comment]
No activity, closing this PR.

**Pull request closed without merging**